environment/src/main/scala/ru/primetalk/funtik/environment/geom2d/Geom2dUtils.scala

```diff
@@ -0,0 +1,144 @@
+package ru.primetalk.funtik.environment.geom2d
+
+
+trait Geom2dUtils[V] extends Vector2dSyntax[V] {
```

Comment:
I don't know how to explain this feeling, but i think that this class is is too complex and his methods, classes and variables has low cohesion.